Lib/mutatorMath/objects/mutator.py

```diff
@@ -133,7 +133,7 @@ def getAxisNames(self):
         s = {}
         for l, x in self.items():
             s.update(dict.fromkeys([k for k, v in l], None))
-        return s.keys()
+        return sorted(list(s.keys()))
```

Comment:
so you're saying, for "robustness" one should never use the regular `dict` object and always prefer an OrderedDict? (on a side note, the feature of python3.6 dictionary keeping its keys' order is just an implementation detail, and one should not rely on this when writing portable python)